scripts/backCompatChecker.py

```diff
@@ -0,0 +1,115 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import re
+import os
+import subprocess
+
+oldArguments = []
+newArguments = []
+
+class Arguments(dict):
+    def __init__(self, command, name, isRequired):
+        self.command = command
+        self.name = name
+        self.isRequired = isRequired
+        dict.__init__(self,command = command, name = name, isRequired = isRequired)
+
+def extractArgumentsFromCommand(command):
+    print('running extractArgumentsFromCommand for ' + command)
+    argumentList = []
+    commandExtended = 'az ' + command + ' -h'
+    help_text = subprocess.run(commandExtended.split(' '), shell=True, stdout=subprocess.PIPE)
+    print('help text for ' + command)
+    print(help_text)
```

Comment:
You can remove these print statements

Reply:
keeping this because it will help with debugging, specially with help text from released extension so that developer will not have to do that manually again